README.md

```diff
@@ -1,102 +1,97 @@
-augur-core
-----------
+# augur-core
 
 [![Build Status](https://travis-ci.org/AugurProject/augur-core.svg)](https://travis-ci.org/AugurProject/augur-core)
 
-### Ethereum contracts for a decentralized prediction market platform
+Ethereum contracts for a decentralized prediction market platform.
 
-Depends on Serpent and Pyethereum
+## Installation
 
-# Installation
+You should already have a system-wide installation of Python and it should be
+Python 2.7.
 
-You should already have a system-wide installation of python, and it should be set to python-2.7.x.
+First install the dependencies, which include PyEthereum (the tool used to test
+Ethereum smart contracts from Python scripts) and the Serpent smart contract
+programming language:
 
-First set up pyethereum, which includes the tools used to test Ethereum smart contracts from python scripts, and the serpent smart contract programming language. <br>
-(can be done as your preferred user from your home directory)
-
-Pyethereum & Serpent Installation:
 ```
 sudo pip install -r requirements.txt
 ```
 
+(If installed outside of a Python virtualenv Serpent will be installed to
+`./src/serpent`. This isn't ideal but won't hurt anything.)
```

Comment:
this is a byproduct of the installation method and not a fault with the ethereum-serpent package itself; since there's not a recent version we install with `pip install -e` and use a `git+https` URL. `-e` means `editable` and that's why it ends up in `./src` if you're not inside a virtualenv.

there are a few ways to fix:

- wait for ethereum-serpent to make a new PyPi release (hasn't happened since 2015)
- upload our own release of ethereum-serpent to PyPi and point to that until they make their own (not unprecedented)
- advise people to install with `pip install -r requirements.txt --src ./.deps` (installs to `.deps` instead of `src`)
- advise people to use a virtualenv (but this doesn't help with installation inside docker)